manager/backend/user_manager/mgmt_token_manager.py

```diff
@@ -0,0 +1,156 @@
+# or (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing, please email
+# opensource@seagate.com or cortx-questions@seagate.com.
+
+import jwt
+import uuid
+from datetime import datetime, timedelta
+from enum import Enum
+from manager.backend.user_manager.error import (MgmtExpiredTokenError,
+                                                MgmtInvalidTokenError)
+
+
+class JWTConst(Enum):
+    """Constants defined for JWT."""
+    JWT_ALGORITHM = 'HS256'
+    JWT_EXP_DELTA_SECONDS = 90
+    JWT_EXP_DELTA_DAYS = 1
+
+
+class MgmtTokenManager:
+
+    def __init__(self) -> None:
+        """Init Method."""
+
+    def _encode_token(self, payload, token_type, secret):
+        """Encrypt token.
+
+        Args:
+            payload (dict): Information to be included in token.
+            token_type (str): Type of token. Ex. access or refresh token.
+            secret (key): Secret key for token.
+
+        Returns:
+            str: Token.
+        """
+        to_encode = payload.copy()
+        to_encode['token_type'] = token_type
+        if type == "access_token":
+            to_encode.update({
+                "exp": datetime.utcnow() +
+                timedelta(seconds=JWTConst.JWT_EXP_DELTA_SECONDS.value)})
+        else:
+            to_encode.update({
+                "exp": datetime.utcnow() +
+                timedelta(days=JWTConst.JWT_EXP_DELTA_DAYS.value)})
+
+        return jwt.encode(to_encode, secret,
+                          algorithm=JWTConst.JWT_ALGORITHM.value)
+
+    def create_tokens(self, payload, secret):
+        """Create access and refresh token.
+
+        Args:
+            payload (dict): Information to be included in token.
+            secret (key): Secret key for token.
+
+        Returns:
+            dict: Access and refresh token.
+        """
+        access_token = self._encode_token(payload, "access_token", secret)
+        refresh_token = self._encode_token(payload, "refresh_token", secret)
+
+        tokens = dict(
+            access_token=f"{access_token}",
+            refresh_token=f"{refresh_token}"
+            )
+        return tokens
+
+    def decode_access_token(self, token, secret):
+        """Decrypt access token.
+
+        Args:
+            token (str): Access token.
+            secret (key): Secret key for token.
+
+        Raises:
+            MgmtInvalidTokenError: Invalid access token.
+            MgmtExpiredTokenError: Expired token.
+
+        Returns:
+            dict: Payload information.
```

Comment:
do not return a complete payload just return the Session object.
create Session object from payload inside this function only.

Reply:
This is internal method.
From validate_token we're returning session object only.